File: src/Eredua/Erabiltzailea.java
package Eredua;

public class Erabiltzailea{
    private String NAN;
    private String izena;
    private String abizena;
    private String pasahitza;
    private String rol;
    private boolean ezabatuta;

    public Erabiltzailea(String pNAN, String pIzena, String pAbizena, String pPasahitza, String pRol){
        this.NAN = pNAN;
        this.izena = pIzena;
        this.abizena = pAbizena;
        this.pasahitza = pPasahitza;
        this.rol = pRol;
        this.ezabatuta = false;
    }
}

File: src/Eredua/Erabiltzailea.py


class Erabiltzailea:
    def __init__(self, nan, izena, abizena, pasahitza,):
        self.nan = nan
        self.izena = izena
        self.abizena = abizena
        self.pasahitza = pasahitza
        self.rol = False #Erabiltzailea (True=Admin)
        self.onartu = False
        self.onartuID = None

    